chain/network/src/types.rs

```diff
@@ -363,7 +363,8 @@ pub struct NetworkInfo {
     pub received_bytes_per_sec: u64,
     /// Accounts of known block and chunk producers from routing table.
     pub known_producers: Vec<KnownProducer>,
-    pub tier1_accounts: Vec<Arc<SignedAccountData>>,
+    pub tier1_accounts_data: Vec<Arc<SignedAccountData>>,
```

Comment:
add comments (and also add the comment above next to connected_peers to mention that these are tier2 only)

Reply:
done